prefect_census/client.py

```diff
@@ -0,0 +1,94 @@
+"""Module containing client for interacting with the Census API"""
+from httpx import AsyncClient, Response
+
+
+class CensusClient:
+    """
+    Client for interacting with the Census API.
+
+    Args:
+        api_key: API key to authenticate with the Census API.
+    """
+
+    def __init__(self, api_key: str):
+        self._closed = False
+        self._started = False
+
+        self.client = AsyncClient(
+            base_url="https://app.getcensus.com",
+            headers={"Authorization": f"Bearer {api_key}"},
+        )
+
+    async def call_endpoint(
```

Comment:
Should these async methods be wrapped with `@sync_compatible` so that if users use them, they can be called synchronously (without await)

Reply:
Sure I can do that.